addons/mail/static/src/js/composers/basic_composer.js

```diff
@@ -243,6 +268,27 @@ var BasicComposer = Widget.extend({
     _hideEmojis: function () {
         this._$emojisContainer.remove();
     },
+    /**
+     * Making sure that dragging content is external files.
+     * Ignoring other content draging like text.
+     *
+     * @private
+     * @param {DataTransfer} dataTransfer
+     */
+    _isDragSourceExternalFile: function (dataTransfer) {
+        var DragDataType = dataTransfer.types;
+        if (typeof DOMStringList != 'undefined') {
+            if (DragDataType.constructor == DOMStringList) {
+                return DragDataType.contains('Files');
+            }
+        }
+        if (typeof Array != 'undefined'){
```

Comment:
Unnecessary check, because all browsers have built-in `Array` (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array).